sdk/communication/azure-communication-email/azure/communication/email/_email_client.py

```diff
@@ -63,16 +68,86 @@ def from_connection_string(
     @distributed_trace
     def send(
         self,
-        email_message: EmailMessage,
+        email_message: JSON,
         **kwargs
-    ) -> SendEmailResult:
+    ) -> JSON:
+        # cSpell:disable
         """Queues an email message to be sent to one or more recipients.
 
-        :param email_message: The message payload for sending an email.
-        :type email_message: ~azure.communication.email.models.EmailMessage
-        :return: SendEmailResult
-        :rtype: ~azure.communication.email.models.SendEmailResult
+        :param email_message: The message payload for sending an email. Required.
+        :type email_message: JSON
+        :return: JSON object
+        :rtype: JSON
+
+        Example:
+            .. code-block:: python
+
+                # JSON input template you can fill out and use as your body input.
+                email_message = {
+                    "content": {
+                        "subject": "str",  # Subject of the email message. Required.
+                        "html": "str",  # Optional. Html version of the email message.
+                        "plainText": "str"  # Optional. Plain text version of the email
+                          message.
+                    },
+                    "recipients": {
+                        "to": [
+                            {
+                                "email": "str",  # Email address. Required.
+                                "displayName": "str"  # Optional. Email display name.
+                            }
+                        ],
+                        "CC": [
```

Comment:
Does the service treat these as case insensitive?

Reply:
Good catch. The service needs these lowercase, I'll change this.